### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by Creator-singh**

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    box-shadow: 3px 3px 3px 3px var(--lightgrey);
+}
+
+.card-btn{
+    position: absolute;
+    border: 2px solid var(--primary-color);
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    text-align: center;
+    color: white;
```
Comment (Naveenchandar) [inline components/card/card.css:48]: Please use hex code for consistency

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    border: 2px solid var(--sidebar-color);
+    margin: 1rem;
+    max-width: 30rem;
+    height: 45rem;
+    padding: 1rem;
+    border-radius: 5px;
+}
+
+.card-example{
+    display: flex;
```
Comment (Naveenchandar) [inline components/card/card.css:30]: If flex properties are used often, you can move those to utility file.

Comment (Creator-singh) [reply components/card/card.css:30]: I was using utility but one of the MA mentioned that we should refrain from using many classes in a tag. At most 2 or 3 classes is enough so stopped using it.

Comment (Naveenchandar) [reply components/card/card.css:30]: Okay if you've this way of styling, Make sure to follow in all files. 👍

File: components/card/card.css (hunk 1 above)
Comment (Creator-singh) [reply components/card/card.css:48]: okay but using hex code here cannot give the user idea which color is there so didn't used.

**Review by Naveenchandar — APPROVED** [review]
Looks good

**Review by Aditi0505 — APPROVED** [review]
LGTM

**Pull request merged**